tsdb/config.go

```diff
@@ -128,6 +132,13 @@ type Config struct {
 	// Setting series-id-set-cache-size to 0 disables the cache.
 	SeriesIDSetCacheSize int `toml:"series-id-set-cache-size"`
 
+	// SeriesFileMaxConcurrentCompactions is the maximum number of concurrent compactions
+	// that can be running at one time across all shards.  Compactions scheduled to run when the
```

Comment:
nit: the term `shards` doesn't really belong here. A `SeriesFile` is a per-database concept. Perhaps you mean series file _partitions_ rather than shards?